src/components/Carousel/index.test.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import { render } from '../../../test/test.utils'
+import { Carousel } from '.'
+import { CurratedContentCard } from '../CurratedContentCard'
+
+const cards = [
+  {
+    imageUrl:
+      'https://s3-eu-west-1.amazonaws.com/ticketswap-public/public/201610/80afc84b-2091-4359-9fd7-9126d684182c.jpeg',
+    text: 'IFind your oasis!',
+    title: 'Desert festival',
+  },
+  {
+    imageUrl:
+      'https://s3-eu-west-1.amazonaws.com/ticketswap-public/public/201610/80afc84b-2091-4359-9fd7-9126d684182c.jpeg',
+    text: 'Let’s get wild!',
+    title: 'Jungle festival',
+  },
+  {
+    imageUrl:
+      'https://s3-eu-west-1.amazonaws.com/ticketswap-public/public/201610/80afc84b-2091-4359-9fd7-9126d684182c.jpeg',
+    text: 'Swim like a fish!',
+    title: 'Ocean festival',
+  },
+]
+
+describe('Carousel', () => {
+  it('renders without crashing', () => {
+    const { container, getByText } = render(
+      <Carousel
+        id="curratedContentCarousel"
+        title="Our top picks"
+        subtitle="Explore the best events"
+        viewAll="See all"
+        onClickAll={() => window.alert('This is all')}
+        cards={cards.map((card, index) => (
+          <CurratedContentCard
+            key={index}
+            title={card.title}
+            text={card.text}
+            imageUrl={card.imageUrl}
+          />
+        ))}
+      />
+    )
+
+    expect(container.firstChild).toBeInTheDocument()
+    expect(getByText(/Explore the best events/i)).toBeInTheDocument()
+    expect(getByText(/Desert festival/i)).toBeInTheDocument()
+    expect(getByText(/Ocean festival/i)).toBeInTheDocument()
+    expect(getByText(/Jungle festival/i)).toBeInTheDocument()
+  })
+})
```

Comment:
Maybe good to also add tests for controlling the carousel?

Reply:
Like the previous / next you mean? I tried but struggle a bit to check as all card are present in the document but just hidden. Maybe you have some ideas..